model/team_member.go

```diff
@@ -54,6 +60,30 @@ func TeamUnreadFromJson(data io.Reader) *TeamUnread {
 	return o
 }
 
+func TeamMembersWithErrorToTeamMembers(o []*TeamMemberWithError) []*TeamMember {
+	var ret []*TeamMember
+	for _, o := range o {
+		if o.Error == nil {
+			ret = append(ret, o.Member)
+		}
+	}
+	return ret
+}
+
+func TeamMembersWithErrorToJson(o []*TeamMemberWithError) string {
+	if b, err := json.Marshal(o); err != nil {
+		return "[]"
```

Comment:
I know we do this for `TeamMember` but I wonder if it makes sense to return the `error` here if we fail to marshal to string? I can see this being quite frustrating if someone keeps getting `[]` whilst failing to marshal to string, thought?

Reply:
Let's leave it as is. It's a fake edge case, since in reality there should never be an error here...